mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule PrometheusParser.MixProject do
   def project do
     [
       app: :prometheus_parser,
-      version: "0.1.9",
+      version: "0.1.11",
```

Comment:
```suggestion
      version: "0.1.10",
```

Reply:
There is actually a 0.1.10 on hex already, which is why I went up to 0.1.11.